#16441_Return_single_element_from_array (wip, tests not complete) (#16444)
vlang/v: doc/docs.md

* `a.delete_last()` removes the last element
* `a.first()` equivalent to `a[0]`
* `a.last()` equivalent to `a[a.len - 1]`

> * `a.single()` equivalent to `a[0]` but also panics if it contains more that one element

Reviewer: This needs to be improved.  Right now, it says that if the array has more than one element, the routine will panic.

Maybe something like
```suggestion
* `a.single()` equivalent to `a[0]` but also panics if it the element is not unique
```
